src/main/resources/default-lintConfig.xml

```diff
@@ -0,0 +1,139 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I'm gonna keep it the way it is right now since most of suggestions seems pretty minor. We can make another pr/update after we try out the current config and have a better idea of what to change